### PR TITLE
Add spanish translation for Canada Senate Bitcoin talk

### PR DESCRIPTION
**Pull request by BlueeeMoon**

### CODE REVIEW

**Comment by grunch** [comment]
ACK

great job 😄

File: andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
+---
+title: Andreas Antonopolous Senado de Canadá Bitcoin (2014-10-08)
+transcript_by: Bryan Bishop 
+translation_by: Blue Moon
+tags: ['security']
```
Comment (adamjonas) [inline andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md:5]: These tags should be translated as well:
s/security/seguridad/

File: andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+---
+
+Esta es una transcripción de las pruebas presentadas ante el Comité del Senado de Canadá sobre Banca, Comercio y Economía. Aquí hay un [video](https://www.youtube.com/watch?v=xUNGFZDO8mM&t=3m10s). El discurso de apertura se puede encontrar  [aquí](http://antonopoulos.com/2014/10/09/canadian-senate-hearings-my-opening-remarks/).
```
Comment (adamjonas) [inline andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md:8]:
```suggestion
Esta es una transcripción de las pruebas presentadas ante el Comité del Senado de Canadá sobre Banca, Comercio y Economía. Aquí hay un [video](https://www.youtube.com/watch?v=xUNGFZDO8mM&t=3m10s). El discurso de apertura se puede encontrar [aquí](http://antonopoulos.com/2014/10/09/canadian-senate-hearings-my-opening-remarks/).
```

File: andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+Puede aparecer otra transcripción [aquí](http://www.parl.gc.ca/SenCommitteeBusiness/CommitteeTranscripts.aspx?parl=41&ses=2&Language=E&comm_id=3) pero quién sabe.
+
+Se puede encontrar una transcripción adicional [aquí](http://bravenewcoin.com/news/unrevised-floor-transcript/).
+
+informe final: <http://www.parl.gc.ca/Content/SEN/Committee/412/banc/rep/rep12jun15-e.pdf>
```
Comment (adamjonas) [inline andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md:14]: I realize this is lowercase on the original but capitalization would be more consistent

```suggestion
Informe final: <http://www.parl.gc.ca/Content/SEN/Committee/412/banc/rep/rep12jun15-e.pdf>
```

**Comment by adamjonas** [comment]
Thanks, @BlueeeMoon! A few (nice to have) suggestions for fixups. This very good as is.

File: andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md (hunk 1 above)
Comment (BlueeeMoon) [reply andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md]: The label has been translated.

File: andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md (hunk 2 above)
Comment (BlueeeMoon) [reply andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md]: Blank space has been removed.

**Pull request merged**